site/content/en/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -26,6 +26,15 @@ Make sure the following conditions are met:
 
 VisibilityOnDemand is an `Alpha` feature disabled by default. To use the visibility API  change [the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) and set `VisibilityOnDemand=true`.
 
+### Configure API Priority and Fairness:
```

Comment:
I think that it should be better to mention the APF in the installation section as well: https://kueue.sigs.k8s.io/docs/installation/

Because this feature is now enabled by default and it means that the visibility endpoints will be hosted without use action.

Reply:
Done